prime-dotnet-webapi/ViewModels/Parties/AuthorizedUserChangeModel.cs

```diff
@@ -0,0 +1,86 @@
+using System.Security.Claims;
+using Prime.Models;
+
+namespace Prime.ViewModels.Parties
+{
+    public class AuthorizedUserChangeModel : IPartyChangeModel
+    {
+        public string PreferredFirstName { get; set; }
+        public string PreferredMiddleName { get; set; }
+        public string PreferredLastName { get; set; }
+        public VerifiedAddress VerifiedAddress { get; set; }
```

Comment:
Thank you for accidentally informing me that we are passing up the `VerifiedAddress` in our other PartyChangeModels rather than mapping or validating against the BC Services Card 😅

Given this is exactly the same way we are doing it in other places, this is _fine for now_ ᵇᵘᵗ ʷᵉ ˢʰᵒᵘˡᵈ ʳᵉᵃˡˡʸ ᶜʰᵃⁿᵍᵉ ᵗʰᶦˢ ᵉᵛᵉʳʸʷʰᵉʳᵉ ᶦⁿ ᵗʰᵉ ᵃᵖᵖ

Reply:
@james-hollinger this is a reminder comment so we revisit this next sprint - @mtpultz